azurelinuxagent/ga/cgroupstelemetry.py

```diff
@@ -17,10 +17,23 @@
 import threading
 
 from azurelinuxagent.common import logger
+from azurelinuxagent.common.event import WALAEventOperation, add_event
 from azurelinuxagent.ga.cgroup import CpuCgroup
 from azurelinuxagent.common.future import ustr
 
 
+def log_cgroup_info(formatted_string, op=WALAEventOperation.CGroupsInfo, send_event=True):
```

Comment:
this should probably be in CgroupsApi. CgroupsTelemetry handles collecting the resource metrics